docs/installation.md

```diff
@@ -204,88 +201,7 @@ or exclude namespaces for Prometheus Operator installed with Sumo Logic Kubernet
 
 ### Examples
 
-#### Minimal configuration
-
-An example file with the minimum configuration is provided below.
-
-```yaml
-sumologic:
-  accessId: ${SUMO_ACCESS_ID}
-  accessKey: ${SUMO_ACCESS_KEY}
-  clusterName: ${MY_CLUSTER_NAME}
-```
-
```

Comment:
I think I would actually keep this section and then after it wrote something like "for more configuration examples see this file"